bundles/go/go_completer.moon

```diff
@@ -0,0 +1,50 @@
+import app, config, interact from howl
+import ProcessBuffer from howl.ui
+import Process from howl.io
+import Matcher from howl.util
+append = table.insert
+
+install_gocode = ->
```

Comment:
Instead of doing lifting ourselves here we could re-use the command module. E.g.:

```moonscript
howl.command.exec nil, 'go get -u -v github.com/nsf/gocode'
```
The first argument is the working directory, so we could the current file's directory if we want to.